cmd/controller/app/server.go

```diff
@@ -70,24 +75,24 @@ func Run(s *ServerRunOptions) error {
 	pgInformer := pgInformerFactory.Scheduling().V1alpha1().PodGroups()
 
 	informerFactory := informers.NewSharedInformerFactoryWithOptions(kubeClient, 0, informers.WithTweakListOptions(func(opt *metav1.ListOptions) {
-		opt.LabelSelector = util.PodGroupLabel
 	}))
 	podInformer := informerFactory.Core().V1().Pods()
-	ctrl := controller.NewPodGroupController(kubeClient, pgInformer, podInformer, pgClient)
-	pgInformerFactory.Start(stopCh)
-	informerFactory.Start(stopCh)
+	pgCtrl := controller.NewPodGroupController(kubeClient, pgInformer, podInformer, pgClient)
+	eqCtrl := initEQController(kubeClient, pgInformerFactory, pgClient, podInformer)
```

Comment:
rename `pgInformerFactory`  may be better since this will be used in ElasticQuota Controller